src/components/progress/base.ts

```diff
@@ -116,5 +141,9 @@ export abstract class IgcProgressBaseComponent extends LitElement {
     this.requestUpdate();
   }
 
-  protected abstract runAnimation(start: number, end: number): void;
+  protected abstract runAnimation(
+    start: number,
+    end: number,
+    indeterminateChange?: boolean
```

Comment:
You can already get `this.indeterminate` inside the method call. 
But why do we need this at all? Can't we manage the label update based on the state of the animation object?